.github/workflows/e2e-jimm-k8s-charm-oidc-auth.yml

```
@@ -25,7 +25,7 @@ jobs:
         node-version: [22.x]
     steps:
       - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
+        uses: Jayllyz/free-disk-space@main
```

```suggestion
        uses: Jayllyz/free-disk-space@fbe203b3788f2bebe2c835a15925da303eaa5efe
```

Could potentially pin this action to a commit since it's not super popular

---

Thanks.